bin/convert-argv.js

```diff
@@ -35,11 +35,11 @@ module.exports = function(optimist, argv, convertOptions) {
 	}
 
 	var configFileLoaded = false;
-	var configPath, ext;
+	var configFile = {};
```

Comment:
could you write in a way in which `configFile` (or `configFiles`) is always an array? (why? see :one: marker)